app/controllers/users_controller.rb

```diff
@@ -97,13 +97,19 @@ def show
 
   # GET /facilities/:facility_id/users/:user_id/access_list
   def access_list
+    # Unsupported in cross-facility mode
+    raise ActiveRecord::RecordNotFound if current_facility.cross_facility?
```

Comment:
I chose to do this in the controller rather than the Ability since I want it entirely disabled rather than just disabled for some roles.